receiver/awss3receiver/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+type: awss3
+
+status:
+  class: receiver
+  stability:
+    development: [traces]
+  distributions: [contrib]
```

Comment:
Should this be considered a part of the `contrib` distribution, even if it's not included in releases?

Reply:
@crobert-1 is this a question for me? I suspect this was just copy'n'pasted from another receiver, happy to be told what to put here.

Reply:
I would leave the `distributions` field empty for now